### PR TITLE
Add exploratory charter issue template

### LINKED ISSUE #26: Add issue template for exploratory charter

### PR DESCRIPTION
**Pull request by hartsick**
On exploratory testing and charters: https://medium.com/product-labs/charters-exploratory-testing-and-beyond-87315184f256

Closes #26

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/exploratory-charter.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+name: Exploratory Charter
+about: Suggest a structured exploration
+title: '[Explore] <title>'
+labels: research
```
Comment (hartsick) [inline .github/ISSUE_TEMPLATE/exploratory-charter.md:5]: I added the `research` label here, does that seem right?

File: .github/ISSUE_TEMPLATE/exploratory-charter.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+assignees: ''
+
+---
+
+## Charter
+
+**Explore** [target]
+**With** [resources]
+**To discover** [information]
```
Comment (rahearn) [inline .github/ISSUE_TEMPLATE/exploratory-charter.md:14]: Would a link to the blog post describing this format be useful here? I know that helped me a lot on another project where we were using [Conventional Comments](https://conventionalcomments.org/) to remember exactly what they meant each time I did a review.

Comment (hartsick) [reply .github/ISSUE_TEMPLATE/exploratory-charter.md:14]: Yes, definitely! Will add.

**Review by rahearn — APPROVED** [review]
🎉

**Pull request merged**